.travis.yml

```diff
@@ -9,6 +9,7 @@ before_script:
     - psql -c 'create database littlewindowtest;' -U postgres
 script:
   - npm run lint
-  # TODO: Add in tests!
-  # - npm test --passWithNoTests
-  # - cd client && npm test --passWithNoTests
+  - npm run testtravis
+  - cd client && npm install
+  - npm run test --passWithNoTests
```

Comment:
It looks like `--passWithNoTests` is [already set in the `package.json`](https://github.com/chaynHQ/little-window/pull/250/files#diff-52dbbf94d5c54b68c0cad0702ba94a96R22) so probably don't need it here.

Reply:
Update Made